LogIn/Helpers/SwiftUI+KeyboardFrameModifier.swift

```diff
@@ -0,0 +1,45 @@
+//
+//  SwiftUI+KeyboardFrameModifier.swift
+//  LogIn
+//
+//  Created by Aleksei Permiakov on 07.02.2022.
+//
+
+import SwiftUI
+import Combine
+
+struct AdaptsToKeyboard: ViewModifier {
+    @State var currentHeight: CGFloat = 0
```

Comment:
Still state in the view modifier is a bad idea

Reply:
We probably need to discuss why is it a bad idea.